tests/components/huawei_lte/test_config_flow.py

```diff
@@ -235,19 +245,36 @@ async def test_success(hass, login_requests_mock):
                 "errors": {},
             },
         ),
+        (
+            {
+                "method": ANY,
+                "url": f"{FIXTURE_USER_INPUT[CONF_URL]}api/device/basic_information",
+                "exc": Exception("Something unexpected"),
+            },
+            {
+                # Does not matter
+            },
+            {
+                "type": data_entry_flow.FlowResultType.ABORT,
+                "reason": "unsupported_device",
+            },
+        ),
     ),
 )
-async def test_ssdp(hass, upnp_data, expected_result):
+async def test_ssdp(
+    hass, login_requests_mock, requests_mock_request_kwargs, upnp_data, expected_result
+):
     """Test SSDP discovery initiates config properly."""
-    url = "http://192.168.100.1/"
+    url = FIXTURE_USER_INPUT[CONF_URL][:-1]  # strip trailing slash for appending port
     context = {"source": config_entries.SOURCE_SSDP}
+    login_requests_mock.request(**requests_mock_request_kwargs)
```

Comment:
I thought I had the tests passing locally yesterday, but apparently messed something up. Anyway there's something wrong with the request mockery setup here, getting `NoMockAddress` for `POST http://192.168.1.1/api/user/login`, will get back to this. We aren't even supposed to be logging in here, so perhaps use a more/better dedicated/set up requests_mock.

Reply:
Incidentally, we just require an upgraded huawei-lte-api (which brings in fixes to other things) to fix the tests. Will upgrade the toplevel PR description.